sfaira/data/dataloaders/obs_utils.py

```diff
@@ -257,6 +262,16 @@ def streamline_obs_uns(adata: anndata.AnnData,
     # Add additional hard-coded description changes for cellxgene schema:
     if schema.startswith("cellxgene"):
         schema_version = schema.split(":")[-1] if ":" in schema else None
+        # convert all custom ontology symbols to unknown symbols as custom labels are not supported in the cellxgene schema
+        for k in [
```

Comment:
1. Maybe check first if custom terms appear at all in .obs columns before rewriting these, i think there might be some edge cases like categoricals where rewriting might cause some problems?
2. I would also change `# convert all custom ontology symbols to unknown symbols as custom labels are not supported in the cellxgene schema` to `# convert all custom ontology symbols to unknown symbols in ontology constrained columns so that these values can be fully handled by ontology operations` as this is not cellxgene specific